scriptmodules/emulators/quasi88.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="quasi88"
+rp_module_desc="NEC PC-8801 emulator"
+rp_module_help="ROM Extensions: .d88 .D88 .cmt .CMT .t88 .T88."
+rp_module_section="exp"
+version="0.6.4"
+
+function depends_quasi88() {
+    local depends=(libsdl1.2-dev)
+    getDepends "${depends[@]}"
```

Comment:
you can just do `getDepends libsdl1.2-dev` in this case, but it's not a problem.